plugins/woocommerce/client/admin/client/settings-payments/components/payment-gateways/payment-gateways.tsx

```diff
@@ -1,10 +1,10 @@
 /**
  * External dependencies
  */
+import { useState } from 'react';
```

Comment:
shouldn't we be using wordpress/element library for that?

Reply:
We use both in the codebase, but it’s better to keep it consistent. I’ve updated the import to use `@wordpress/element`.